tfx/components/example_validator/component.py

```diff
@@ -91,10 +97,25 @@ def __init__(self,
           'been renamed to "statistics" and is deprecated. Please update your '
           'usage as support for this argument will be removed soon.')
       statistics = stats
-    anomalies = output or types.Channel(
-        type=standard_artifacts.ExampleAnomalies,
-        artifacts=[standard_artifacts.ExampleAnomalies()])
+    if exclude_splits is None:
+      exclude_splits = ['train']
```

Comment:
print an info log like "Exclude `train` split when exclude_splits is unset (None)"